src/main/java/edu/njit/jerse/ashe/llm/chatgpt/GptApiClient.java

```diff
@@ -1,12 +1,13 @@
-package edu.njit.jerse.ashe.services;
+package edu.njit.jerse.ashe.llm.chatgpt;
```

Comment:
Nit: I'd prefer `openai` or `gpt` for the package name, personally. But, I might be [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality).

Reply:
I agree - change will be made